index.js

```diff
@@ -42,6 +46,7 @@ module.exports = {
     "max-line-length": [ 80, {
       ignore: "non-comments",
     } ],
+    "max-nesting-depth": 3,
```

Comment:
imho we should roll our own `suitcss-nesting` plugin to allow only suit compliant nested rules. Anyway I think that in [the worst case](https://gist.github.com/giuseppeg/3db09a8a7a0046e813d0923dc4393144) the dept is `3`, but I might be wrong.